x/incentives/spec/02_state.md

```diff
@@ -0,0 +1,58 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Incentives management
+
+All the incentives that is going to be provided are locked into `IncentivePool` until it's released to the users after a specific period of time.
+
+### Pot
+
+A `Pot` is a unit of incentive distributer. It describes the the users who can get reward, and the amount of coins to be distribute, the distribution start time, and the number of epochs it provide incentive from the `Pot`.
```

Comment:
```suggestion
Rewards to be distributed are organized by `Pot`. The `Pot` describes how users can get reward, stores the amount of coins in the pot, the cadence at which rewards are to be distributed, and the number of epochs to distribute the reward over.
```

Reply:
I'm generally confused on the mental model of this. Is the idea that one AMM creates a single pot, and then every LP'r adds a new distr condition to the pool's Pot via the pool's API?

Reply:
Or is the idea that everyone who has locked up tokens gets their own pot? (The current API seems to reflect that)
I don't think thats ideal, since it causes much more state / computational overhead, and this is actually going to be a bottleneck for our system. (Also I think we can improve debuggability by having a per-AMM pot)